data/Slovakia/National_Council/unstable/stats.json

```diff
@@ -9,7 +9,7 @@
   },
   "terms": {
     "count": 5,
-    "latest": "2012-03-11"
+    "latest": ""
```

Comment:
We can't let this go through with this. It turns out that this is a wider problem — https://github.com/everypolitician/everypolitician/issues/537. But this is going to have to wait on that being fixed first.